src/cli/expand-patterns.js

```diff
@@ -4,6 +4,10 @@ import { fastGlob } from "./prettier-internal.js";
 
 /** @typedef {import('./context').Context} Context */
 
+function isError(pathOrError) {
+  return Object.hasOwn(pathOrError, "error");
+}
```

Comment:
It should be safe to check `.error` directly.

Reply:
I didn't mean to use `in`, I mean it can be removed. Do we still need it?

Reply:
no, I just misunderstood your comment.